### PR TITLE
[kjobctl] Fix crash issue in Slurm mode when ntasks > 1

### PR DESCRIPTION
**Pull request by IrvingMg**
#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:

It fixes a crash issue when the Slurm template has one container and --ntasks is greater than 1.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: cmd/experimental/kjobctl/pkg/builder/slurm_builder.go [hunk 1]
```diff
@@ -412,7 +412,7 @@ func (b *slurmBuilder) build(ctx context.Context) (runtime.Object, []runtime.Obj
 			}
 		}
 
-		for i, j := 0, 0; i <= int(nTasks); i++ {
+		for i, j := 0, 0; i <= len(job.Spec.Template.Spec.Containers) && j < int(nTasks); i++ {
```
Comment (mbobrovskyi) [inline cmd/experimental/kjobctl/pkg/builder/slurm_builder.go:415]: Do we need a this loop? Can we combine them into one?

File: cmd/experimental/kjobctl/pkg/builder/slurm_builder.go
Comment (mbobrovskyi) [inline cmd/experimental/kjobctl/pkg/builder/slurm_builder.go:1]: I have another scenario: What happens if the container length is zero?

Comment (mbobrovskyi) [reply cmd/experimental/kjobctl/pkg/builder/slurm_builder.go:1]: Probably, we can fix it in another PR.

File: cmd/experimental/kjobctl/pkg/builder/slurm_builder.go (hunk 1 above)
Comment (IrvingMg) [reply cmd/experimental/kjobctl/pkg/builder/slurm_builder.go]: Done. Thanks!

File: cmd/experimental/kjobctl/pkg/builder/slurm_builder.go
Comment (IrvingMg) [reply cmd/experimental/kjobctl/pkg/builder/slurm_builder.go:1]: If containers length is zero, that would mean that the template has no containers, right? Is that even a valid case?

**Review by mbobrovskyi — COMMENTED** [review]
/lgtm

Thank you so much!

File: cmd/experimental/kjobctl/pkg/builder/slurm_builder.go
Comment (mbobrovskyi) [reply cmd/experimental/kjobctl/pkg/builder/slurm_builder.go:1]: Ah, you right, we can't create template without containers.

**Review by trasc — COMMENTED** [review]
/approve

**Pull request merged**